scripts/create_tables.sql

```diff
@@ -1 +1,24 @@
 -- create doctors, patients, and appointments tables
+CREATE TABLE doctors(surname,given_name,gender,born_on,specialty)
+  id SERIAL PRIMARY KEY,
+  surname TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on DATE
+  specialty TEXT,
+);
+
+CREATE TABLE patients(surname,given_name,gender,born_on,insurance)
+  id SERIAL PRIMARY KEY,
+  surname TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on DATE
+  insurance TEXT,
+);
+
+CREATE TABLE appointments(date, time, comment)
+id SERIAL PRIMARY KEY,
+date DATE,
+time TEXT,
+comment TEXT,
```

Comment:
you are missing a few lines for the appointments table. Recall that we want each appointment to be linked to a doctor and a patient, by using their ids. Take a look at this solution:

```sql
CREATE TABLE doctors(
  id SERIAL PRIMARY KEY,
  surname TEXT,
  given_name TEXT,
  gender TEXT,
  dob TEXT,
  specialty TEXT
);
-- create patients
CREATE TABLE patients(
  id SERIAL PRIMARY KEY,
  surname TEXT,
  given_name TEXT,
  gender TEXT,
  dob TEXT,
  insurance TEXT
);
-- create appointments
CREATE TABLE appointments(
  date TEXT, time TEXT, comment TEXT,
  doctor_id INTEGER REFERENCES doctors(id),
  patient_id INTEGER REFERENCES patients(id)
);
```